events/blocks/event-subscription-form/event-subscription-form.js

```diff
@@ -46,11 +47,11 @@ async function handleSubmit(event, bp) {
   // Validate the input
   const email = inputElement.value;
   if (email.length === 0) {
-    console.log('Email is a required Field');
+    window.lana?.log('Email is a required Field');
```

Comment:
Hmmmm I don't think we should use Lana to log when the user makes a mistake. It'll blow up our FE Splunk log for nothing. It's only for things we need to debug.